workers/unity/Assets/Gdk/Core/Logging/FieldName.cs

```diff
@@ -0,0 +1,9 @@
+namespace Improbable.Gdk.Core
+{
+    public static class FieldName
```

Comment:
Please get rid of this. You can't tell at a glance what this is or what it relates to. However, I'm happy for there to be a set of static logging helper functions that extracts the relevant information if it is the right stored as the right type in the dictionary.